src/index.js

```diff
@@ -169,6 +184,31 @@ export default class AuthnWidget {
     }
   }
 
+  postExternalAuthenticationCompleted() {
+    setTimeout(() => {
```

Comment:
This seems like it could have reuse. Maybe calling it postContinueAuthentication?